src/main/index.js

```diff
@@ -154,7 +154,9 @@ app.on("will-quit", function () {
   console.log("will-quit event being handled"); // allow-log
   // mainWindow = null;
 
-  axios.get(`http://localhost:${flask_port}/shutdown`);
+  axios.get(
+    `http://localhost:${flask_port}/shutdown?called_through_app_will_quit=true`
```

Comment:
On the python flask server end the new query parameter 'called_through_app_will_quit', is it handled to shutdown ?

Reply:
Flask allows you to pass extra unhandled parameters without having an effect. This will just help with logging, hopefully to help troubleshoot the issue one of our beta customers is having.  Similar to how when the frontend calls `/system_status`, it includes the `current_vuex_status_uuid` as a parameter---the backend doesn't use that, it's just for being able to debug better using the logs.